tests/conftest.py

```diff
@@ -15,3 +23,45 @@ def vcr_config():
     return {
         "filter_headers": [CROSSREF_HEADER_KEY, SEMANTIC_SCHOLAR_HEADER_KEY],
     }
+
+
+@pytest.fixture()
+def unique_tmp_subfolder(tmp_path):
+    yield tmp_path
+    # Cleanup after the test
+    if tmp_path.exists():
+        shutil.rmtree(tmp_path, ignore_errors=True)
+
+
+@pytest.fixture()
+def agent_module_dir(unique_tmp_subfolder, monkeypatch):
+    """Set up a unique temporary folder for the agent module."""
+    monkeypatch.setenv("PQA_HOME", str(unique_tmp_subfolder))
+    importlib.reload(paperqa.agents)
+    return unique_tmp_subfolder
+
+
+@pytest.fixture()
+def agent_index_dir(agent_module_dir):
+    return str(agent_module_dir / ".pqa/indexes")
+
+
+@pytest.fixture(name="agent_test_kit")
+def fixture_agent_test_kit() -> Answer:
+    return Answer(question="What is is a self-explanatory model?")
+
+
+@pytest.fixture(name="stub_paper_path_with_details", scope="session")
+def fixture_stub_paper_path_with_details() -> tuple[Path, dict]:
+    # Corresponds with https://www.semanticscholar.org/paper/A-Perspective-on-Explanations-of-Molecular-Models-Wellawatte-Gandhi/1db1bde653658ec9b30858ae14650b8f9c9d438b
+    return (
+        Path(__file__).parent / "paper.pdf",
+        {
```

Comment:
Wdyt of returning `PaperDetail`s object here instead of metadata?

Reply:
I was keeping the details for posterity, but it's not even used anymore, I'm just going to cut it